airflow/ui/src/context/openGroups/OpenGroupsProvider.tsx

```diff
@@ -46,9 +48,22 @@ export const OpenGroupsProvider = ({ children, dagId }: Props) => {
     [openGroupIds, setOpenGroupIds],
   );
 
+  const expandAllGroups = useCallback(
+    (allGroupIds: Array<string>) => {
+      const uniqueGroupIds = [...new Set([...openGroupIds, ...allGroupIds])];
```

Comment:
```suggestion
      const uniqueGroupIds = [...new Set([...allGroupIds])];
```

Reply:
Actually I think we are ok with `const uniqueGroupIds = [...new Set(allGroupIds)];`